src/lib.rs

```diff
@@ -76,8 +91,13 @@ impl DerefMut for Pwm {
     }
 }
 
+/// The `Pwm` struct represents a PWM (Pulse Width Modulation) controller with a PCA9685 chip and it's
+/// output enable pin.
+///
 pub struct Pwm {
     pca: Pca9685<I2cdev>,
+    /// * `oe_pin`: The `oe_pin` component is a pin that is used to enable or disable the output of the PWM
+    /// signal. It is connected to the Output Enable (OE) pin of the PCA9685 PWM controller.
```

Comment:
It's missing the state where the PCA is enabled.

Reply:
added to docs